### PR TITLE
Improve logs on IpmitoolOutOfBandManagementDriver

### PR DESCRIPTION
**Pull request by GutoVeronezi**
### Description
This PR intends to improve logging in the class `IpmitoolOutOfBandManagementDriver` to facilitate troubleshooting.

### Types of changes
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functioanlity)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale
- [ ] Major
- [x] Minor

### How Has This Been Tested?
It has been tested locally on a test lab.

### CODE REVIEW

File: plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java [hunk 1]
```diff
@@ -37,6 +37,7 @@
 import java.util.List;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import org.apache.commons.lang3.StringUtils;
```
Comment (DaanHoogland) [inline plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java:40]: please use our own (cloudstack) Stringutils as proxy, even when calling the apache.commons

**Review by DaanHoogland — APPROVED** [review]
code looks good

**Review by GabrielBrascher — APPROVED** [review]
Code LGTM.

File: plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java (hunk 1 above)
Comment (rafaelweingartner) [reply plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java:40]: There is no consensus on that. I would prefer this way, using the library directly. We do not get benefits on a facade like that.

Comment (DaanHoogland) [reply plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java:40]: yes we do, and we are badly hampered in most cases where we don't (gson/logging mostly)

Comment (rafaelweingartner) [reply plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java:40]: So, lets add a facade to Spring as well, and to any other utils library that we use :)

I still see no benefits on creating such facades.

**Review by rhtyd — APPROVED** [review]

File: plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java (hunk 1 above)
Comment (DaanHoogland) [reply plugins/outofbandmanagement-drivers/ipmitool/src/main/java/org/apache/cloudstack/outofbandmanagement/driver/ipmitool/IpmitoolOutOfBandManagementDriver.java:40]: that is a lazy exaggeration, we have a facade for string utils and we are hampered by several 3rd party libraries for which we don't. If there are resources for making such a thing for spring, great. Let's do it.

BTW, in this case I didn't :-1: on this issue, but on other counts I might as well. duckduck "using 3rd party library facades", will give you some wisdom on this.

out

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by GutoVeronezi** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Re-kicking travis

**Comment by GutoVeronezi** [comment]
@rhtyd @GabrielBrascher @DaanHoogland is there anything else to do?

**Comment by DaanHoogland** [comment]
that's a no

**Pull request merged**